### PR TITLE
Add recursive directory searching. Skip non-text files.

### PR DESCRIPTION
**Pull request by Laharah**
Adds support for finding published notes in folders within the vault by making `fs.get_files_in_dir` a recursive search.

I also added some error handling so that image files etc. are skipped over.

### CODE REVIEW

File: src/lib/fs.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 def change_ext(ext: str, file_path: str):
     """Changes extension in path"""
     pre, _ = os.path.splitext(file_path)
-    return pre + '.' + ext
+    return pre + "." + ext
```
Comment (A) [inline src/lib/fs.py:12]: Seems like I forgot to add `black` to format code

**Comment by A** [comment]
Thanks for this PR! Would be great if you can add a snapshot test for it in a manner like https://github.com/A/obsidian-blog/blob/master/tests/obsidian/page_test.py#L14

**Comment by Laharah** [comment]
Added the test. sorry about the force push if you got hit by it, I accidentally stashed some changes and forgot to add them back.

**Comment by A** [comment]
Thanks much! Merged and published under `0.2.1` version

File: src/lib/fs.py (hunk 1 above)
Comment (moahmedali281-collab) [reply src/lib/fs.py:12]: Got it ✅ 
Here is the *English version only* for your company sheet. No Arabic.

Copy this into `Sheet1` in Excel / Google Sheets:

### *QRUSH HARMONY INC - Task Sheet*
Title	Assignees	Status	Target date
Activate email ahmedalshikh1234@davincivirtual.com with Davinci	You	To Do	2026-04-25
Pay subscription for US business address	You	To Do	2026-04-25
Buy domain http://qrushharmony.com	You	To Do	2026-04-26
Open business PayPal account for QRUSH HARMONY INC	You	To Do	2026-04-28
Design company logo	You	To Do	2026-04-29
Create Instagram + Facebook page for the company	You	To Do	2026-04-30
Add 3 products to the store	You	To Do	2026-05-02
Register company with bank	You	To Do	2026-05-05
### *How to use it:*
1.  Paste this table in your sheet
2.  Link the sheet to GitHub Projects 
3.  Change `Status` to `In Progress` when you start, and `Done` when you finish
4. 

Do you want me to create this as a downloadable `.xlsx` Excel file for you?

**Pull request merged**